sky/shell/platform/android/org/domokit/sky/shell/SkyApplication.java

```diff
@@ -82,6 +84,14 @@ public void connectToService(Context context, Core core, MessagePipeHandle pipe)
             }
         });
 
+        registry.register(GcmService.MANAGER.getName(), new ServiceFactory() {
```

Comment:
Done.

It'd be nice if there were a way for us to handle this boilerplate for app developers. Something in flutter_tools that auto-generates the AndroidManifest.xml and relevant Java files required to use these services. The manifest especially is something that can be easily screwed up.